src/main/java/com/wcc/platform/domain/platform/mentorship/MentorshipType.java

```diff
@@ -1,8 +1,24 @@
 package com.wcc.platform.domain.platform.mentorship;
 
+import java.util.Arrays;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
 /* Type of mentorship cycle */
+@Getter
+@AllArgsConstructor
 public enum MentorshipType {
-  AD_HOC,
-  LONG_TERM,
-  STUDY_GROUP
+  AD_HOC(1),
+  LONG_TERM(2),
+  STUDY_GROUP(3);
```

Comment:
We discussed that studyGroup should not be part of Mentorship Types, they are a different type of collaboration between mentors, but it is not directly a mentorship type, probably it would be study group (long, short, workshop) but for now probably you can just remove from this enum.